examples/md-flexible/src/TimeDiscretization.cpp

```diff
@@ -44,9 +44,8 @@ void calculatePositions(autopas::AutoPas<ParticleType> &autoPasContainer,
     const auto distanceMovedSquared = dot(displacement, displacement);
     if (distanceMovedSquared > maxAllowedDistanceMoved) {
 #pragma omp critical
-      std::cerr << "A particle moved farther than skin/2/rebuildFrequency: " << std::sqrt(distanceMovedSquared) << " > "
-                << autoPasContainer.getVerletSkin() << "/2/" << autoPasContainer.getVerletRebuildFrequency() << " = "
-                << maxAllowedDistanceMoved << "\n"
+      std::cerr << "A particle moved farther than verletSkinPerTimestep/2: " << std::sqrt(distanceMovedSquared) << " > "
+                << autoPasContainer.getVerletSkinPerTimestep() << "/2 = " << maxAllowedDistanceMoved << "\n"
```

Comment:
If we change the calculation here, then maybe also above in the actual calculation of `maxAllowedDistanceMoved`?
`const auto maxAllowedDistanceMoved = autoPasContainer.getVerletSkinPerTimestep() / 2`

Reply:
[d991b24](https://github.com/AutoPas/AutoPas/pull/724/commits/d991b242d7d6c4bcb195074e7cc7bc31da94f5c6)